src/components/PersonLink.tsx

```diff
@@ -0,0 +1,20 @@
+import { NavLink } from 'react-router-dom';
+
+export const PersonLink = ({ person, people, name }) => {
+  if (!name) {
+    return '-';
+  }
+
+  const foundPerson = people.find(p => p.name === name);
+
+  return foundPerson ? (
+    <NavLink
+      to={`/people/${person.slug}`}
```

Comment:
The `to` attribute uses `person.slug`, but `person` might be undefined if `foundPerson` is false. Consider using `foundPerson.slug` instead to ensure that the slug is only accessed when the person is found.